src/common/stream.cpp

```diff
@@ -143,7 +143,7 @@ bool SeekableSubReadStream::seek(int32 offset, int whence) {
 	switch (whence) {
 	case SEEK_END:
 		offset = size() + offset;
-		// fallthrough
+		// Fall through
```

Comment:
That was on purpose. One of the compilers, either clang or gcc, in at least one version back in the day recognized "fallthrough" but no other spellings.

(C++17 has "[[fallthrough]]", but this code predates C++17)